src/mlpack/tests/lmnn_test.cpp

```diff
@@ -145,13 +142,13 @@ BOOST_AUTO_TEST_CASE(LMNNInitialEvaluationTest)
   double objective = lmnnfn.Evaluate(arma::eye<arma::mat>(2, 2));
 
   // Result calculated by hand.
-  BOOST_REQUIRE_CLOSE(objective, 9.456, 1e-5);
+  REQUIRE(objective == Approx( 9.456).epsilon(1e-7));
```

Comment:
```suggestion
  REQUIRE(objective == Approx(9.456).epsilon(1e-7));
```

Remove the extra space here.